partition/mbr/partiton.go

```diff
@@ -111,14 +111,16 @@ func (p *Partition) writeContents(f util.File, logicalSectorSize, physicalSector
 	b := make([]byte, physicalSectorSize, physicalSectorSize)
 	// we start at the correct byte location
 	start := p.Start * uint32(logicalSectorSize)
+	size := p.Size * uint32(logicalSectorSize)
```

Comment:
Why these changes too MBR?

Reply:
Let me pull that out. I didn't realize I was making multiple changes, so didn't fork originally and mucked up the commit when I did. These changes are really part of PR #18.